facilities/models.py

```diff
@@ -0,0 +1,58 @@
+from django.db import models
+from common.models import AbstractBase, SubLocation
+
+FACILLITY_STATUS = (
+    ('OPERATIONAL', 'Operations are running normally'),
+    ('NOT_OPERATIONAL', 'The facility is not operating'),
+)
+
+FACILITY_TYPES = (
+    ('DISPENSARY', 'dispensary'),
+    ('HEALTH_CENTER', 'Health Center'),
+    ('LEVEL_1', 'Level 1 facility'),
+    ('LEVEL_2', 'Level 2 facility'),
+    ('LEVEL_3', 'Level 3 facility'),
+    ('LEVEL_4', 'Level 4 facility'),
+    ('LEVEL_5', 'Level 5 facility'),
+)
+
+
+class Owner(AbstractBase):
+    name = models.CharField(max_length=100, unique=True)
+    description = models.TextField(null=True, blank=True)
+    code = models.CharField(max_length=100)
+
+
+class Service(AbstractBase):
+    name = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
+    code = models.CharField(max_length=100, null=True, blank=True)
+
+    def __unicode__(self):
+        return self.name
+
+
+class Facility(AbstractBase):
+    name = models.CharField(max_length=100, unique=True)
+    code = models.CharField(max_length=100, unique=True)
+    latitude = models.CharField(max_length=255)
```

Comment:
I'd prefer to have the GIS stuff abstracted into a separate models e.g `FacilityCoordinates` - with as `OneToOneField` relation back to the `Facility`.

There will be more "adventures" with the co-ordinate data than with the facilities data.